Constrained_triangulation_3/doc/Constrained_triangulation_3/Constrained_triangulation_3.txt

```diff
@@ -0,0 +1,308 @@
+before performing the self-intersection check.
+
+If self-intersections are found, the triangulated mesh is converted into a triangle soup, which can then be
+processed with
+\link CGAL::Polygon_mesh_processing::autorefine_triangle_soup(PointRange&,TriangleRange&,const NamedParameters&) `CGAL::Polygon_mesh_processing::autorefine_triangle_soup()`\endlink
+to resolve the self-intersections.
+
+\cgalExample{Constrained_triangulation_3/ccdt_3_preprocessing.cpp }
+
+\subsection CT_3_example_remesh Remeshing a Conforming Constrained Delaunay Triangulation
+
+After constructing the triangulation, you can improve its quality or adapt it to a specific sizing
+field by applying the
+\link CGAL::tetrahedral_isotropic_remeshing(CGAL::Triangulation_3<Traits, TDS, SLDS>&, const SizingFunction&, const NamedParameters&) `CGAL::tetrahedral_isotropic_remeshing()`\endlink
+ function from the \ref PkgTetrahedralRemeshing package.
+
+The following example demonstrates how to remesh a conforming constrained Delaunay triangulation.
+
+\cgalExample{Constrained_triangulation_3/remesh_constrained_Delaunay_triangulation_3.cpp }
+
+
+\subsection CT_3_examples_figure Figures
+
+The following table of figures (\cgalFigureRef{CT_3_ccdt_examples_fig}) illustrates some results of the examples provided in this package.
+The left column shows the input PLC, while the right column displays the resulting conforming
+constrained Delaunay triangulation.
+
+From top to bottom, the lines show different input data (all from) the same input triangulated surface and,
+for each of them, the resulting conforming constrained Delaunay triangulation.
+The input data are:
+<ul>
+<li> the input PLC with no preprocessing;
+<li> the input PLC with a segmentation of the surface patches done with `CGAL::Polygon_mesh_processing::region_growing_of_planes_on_faces()` as done in example \ref CT_3_example_ccdt_region_growing_fimap;</li>
+<li> the input PLC, remeshed and not triangulated, with a segmentation of the surface patches, done with `CGAL::Polygon_mesh_processing::remesh_planar_patches()` as in example \ref CT_3_example_ccdt_fimap;</li>
+<li> the input PLC, isotropically remeshed, with a segmentation of the surface patches, done with `CGAL::Polygon_mesh_processing::region_growing_of_planes_on_faces()` as in example \ref CT_3_example_ccdt_region_growing_fimap.</li>
+</ul>
+It is interesting to note that the conforming constrained Delaunay triangulations built from the same input PLC, but with different
+preprocessing steps, can yield different results.
+
+
+\cgalFigureAnchor{CT_3_ccdt_examples_fig}
+<center>
+<img src="ccdt_examples.png" style="max-width:50%;"/>
+</center>
+\cgalFigureCaptionBegin{CT_3_ccdt_examples_fig}
+A collection of conforming constrained Delaunay triangulations built from different inputs.
+The left column shows the input PLC, while the right column displays the resulting 3D triangulation.<br>
+\cgalFigureCaptionEnd
+
+
+\section CT_3_design Implementation History
```

Comment:
Note sure it makes sense for now to have this section. It makes sense IMHO when you have several iterations or want to acknowledge a "funding"

Reply:
Then the second author should appear